pkg/apps/gitops.go

```diff
@@ -30,7 +30,7 @@ func (o *GitOpsOptions) AddApp(app string, dir string, version string, repositor
 	options := environments.EnvironmentPullRequestOptions{
 		ConfigGitFn: o.ConfigureGitFn,
 		Gitter:      o.Gitter,
-		ModifyChartFn: environments.CreateAddRequirementFn(app, alias, version,
+		ModifyChartFn: environments.CreateAddRequirementFn(o.Helmer, app, alias, version,
```

Comment:
I try to put things like Helmer at the end of the arg list - so that we pass 'variables' first, then interfaces.